updatebreaches.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const DEBUG = !process.env.PUSH_TO_KINTO;
```

Comment:
I’m not sure if this works. Iirc, cli args are strings, so this:
```sh
PUSH_TO_KINTO=1 # Can be any truth-y value
```

Will set `process.env.PUSH_TO_KINTO` to “1” string and not numeric 1.
But I may need to verify when I get to laptop.

Reply:
That's okay though, we only need it to be truth-y. Any non-empty string would do the trick.

Reply:
Yeah, I think that's fine, as long as nobody gets too confused as to `PUSH_TO_KINTO=0` or `PUSH_TO_KINTO=false` also being truthy values. But since this isn't something for end users and is only for OPs folk, i think it's 👌.

Reply:
I changed the comment to say "non-empty" instead of "truth-y", thanks for pointing this out.